releasenotes/notes/1.1/pauli-apply-layout-cdcbc1bce724a150.yaml

```diff
@@ -10,12 +11,25 @@ features_quantum_info:
 
     .. code-block:: python
 
+=======
+    Added a new :meth:`~.Pauli.apply_layout`  method that is equivalent to
```

Comment:
The difference I see in this file is the `~` in front of `.SparsePauliOp` and `.TranspileLayout`, but I don't see any other difference, like where in `main` with the tabs, maybe someone else made the same tab modification as my PR already..

So if both build the docs correctly (haven't tried without the `~`), both current and incoming would be okei 👍